dpub-aam/role/roles.html

```diff
@@ -60,4 +61,4 @@
 
 </script>
 </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
trying to undo this whitespace-only diff
```suggestion
</html>

```

Reply:
I seem to have made it worse.
```suggestion
</html> 
```

Reply:
The GH interface really fails whitespace diffs.

Reply:
I don't think I can pull my "different whitespace only" commit out of your private branch, but I'm going to r+ and you can take it out or leave it at is...